backend/coreapp/views.py

```diff
@@ -201,3 +216,91 @@ def fork(request, slug):
     )
     new_scratch.save()
     return Response(ScratchSerializer(new_scratch).data, status=status.HTTP_201_CREATED)
+
+@api_view(["GET", "POST"])
+def user_current(request, slug=None):
+    """
+    Get the logged-in user, or sign in with GitHub
+    """
+
+    profile = Profile.objects.filter(id=request.session.get("profile", None)).first()
```

Comment:
I think `request.session.get("profile")` is equivalent to `.get("profile", None)`?

Another option is `Profile.objects.get(id=...)` (which will throw `Profile.DoesNotExist` if it doesn't exist). 

(If `"profile"` is not None and the `Profile` doesn't exist, it might even be worthy of raising an error?)